screensaver.atv4/resources/skins/default/1080i/screensaver-atv4.xml

```diff
@@ -22,7 +31,7 @@
 				<texture>busy.png</texture>
 				<aspectratio>keep</aspectratio>
 				<animation effect="rotate" start="0" end="360" center="55,1015" time="1200" loop="true" condition="true">conditional</animation>
-				<visible>!IsEmpty(Window.Property(screensaver-atv4-loading))</visible>
+				<visible>!String.IsEqual(Window.Property(screensaver-atv4-loading),"false")</visible>
```

Comment:
I believe `` was added to Krypton and up. https://forum.kodi.tv/showthread.php?tid=250862&pid=2281321#pid2281321

Since this is in the helix branch you might want to continue using IsEmpty.

Reply:
nice catch

Reply:
I think I'll just move the addon to krypton